daemon/src/identify/protocol.rs

```diff
@@ -0,0 +1,170 @@
+use crate::Multiaddr;
+use anyhow::Context;
+use anyhow::Result;
+use asynchronous_codec::FramedRead;
+use asynchronous_codec::FramedWrite;
+use asynchronous_codec::JsonCodec;
+use futures::AsyncReadExt;
+use futures::AsyncWriteExt;
+use futures::SinkExt;
+use futures::StreamExt;
+use libp2p_core::identity::PublicKey;
+use serde::Deserialize;
+use serde::Serialize;
+use std::collections::HashSet;
+use std::time::Duration;
+use tokio_extras::FutureExt;
+
+// Start libp2p based protocols from 0.3.0 since the last wire version was 0.2.1
+const PROTOCOL_VERSION: &str = "0.3.0";
```

Comment:
```suggestion
const WIRE_PROTOCOL_VERSION: &str = "0.3.0";
```

without the wire_ prefix it took me a while to undestand the line and the comment, as we're in `identify` module.

Reply:
I would remove the constant and field in `IdentifyMsg` altogether, as discussed [here](https://github.com/itchysats/itchysats/pull/2161#discussion_r916478075).